src/query_parser/v2/parser.y

```diff
@@ -834,6 +841,53 @@ numeric_range(A) ::= LSQB param_num(B) RSQB. [NUMBER]{
   A = NewNumericFilterQueryParam_WithParams(ctx, &B, &B, 1, 1);
 }
 
+/////////////////////////////////////////////////////////////////
+// Numeric Operators
+/////////////////////////////////////////////////////////////////
+
+expr(A) ::= modifier(B) COLON EXCLAMATION EQUAL param_num(C). {
```

Comment:
Consider adding this case as a `numeric_operator` with `EXCLAMATION EQUAL`, instead of having this extra rule for this specific case.

Reply:
If I do that, I'll need to differentiate between the operator that has NOT and the others operators, during the creation of `expr()` from `numeric_operator()`. So, I think that it is better to have this special case for `!=`